src/UIOMatic/Controllers/PetaPocoObjectController.cs

```diff
@@ -357,15 +366,39 @@ public object PostCreate(ExpandoObject objectToCreate)
 
             foreach (var prop in objectToCreate)
             {
+                var propKey = prop.Key;
+                PropertyInfo propI = currentType.GetProperty(propKey);
+                //object value
+                if (propI.GetCustomAttributes().Any(x => x.GetType() == typeof(UIOMaticIgnoreFieldAttribute)))
+                {
+                    switch (propI.Name.ToLower())
```

Comment:
what's the purpose of this switch? Seems like it's related to a specific project implementation?